Sources/ScreenObject/ScreenObject.swift

```diff
@@ -73,13 +65,15 @@ open class ScreenObject {
         self.app = app
         self.expectedElementGetters = [expectedElementGetter]
         self.waitTimeout = waitTimeout
-        try waitForFirstScreenElement()
+        try waitForScreen(firstElementOnly: true)
     }
 
     @discardableResult
-    func waitForScreen() throws -> Self {
+    public func waitForScreen(firstElementOnly: Bool = false) throws -> Self {
+        let testedGetters = firstElementOnly ? [expectedElementGetters.first!] : expectedElementGetters
```

Comment:
Naming nitpick: "tested getters" makes me think they've been tested already, but they haven't at this time.

```diff
-        let testedGetters = firstElementOnly ? [expectedElementGetters.first!] : expectedElementGetters
+        let gettersToTest = firstElementOnly ? [expectedElementGetters.first!] : expectedElementGetters
```